csharp/src/Microsoft.ML.OnnxRuntime/FixedBufferOnnxValue.cs

```diff
@@ -28,7 +28,7 @@ private FixedBufferOnnxValue(MemoryHandle pinnedMemory, OrtValue ortValue, OnnxV
         /// </summary>
         /// <typeparam name="T"></typeparam>
         /// <param name="value"></param>
-        /// <returns></returns>
+        /// <returns>a disposable instance of FixedBufferOnnxValue</returns>
```

Comment:
a little unrelated to this change itself, but a comment describing when we would hit each of the conditional branches below would be nice - if memHandle.HasValue is true => non-string tensors and vice-versa. Capturing this in a  comment below would be nice.